src/org/labkey/test/components/html/BootstrapMenu.java

```diff
@@ -118,6 +118,11 @@ public void clickSubMenu(boolean wait, @LoggedParam String ... subMenuLabels)
         clickSubMenu(wait ? getWrapper().getDefaultWaitForPage() : 0, subMenuLabels);
     }
 
+    public boolean menuItemIsDisabled(String text)
+    {
+        return findDisabledMenuItemOrNull(text) != null;
+    }
```

Comment:
I'm not sure this is quite right. It will return true if the menu item is missing, which is not what I would expect. Consider throwing if the menu item isn't there.

```suggestion
    public boolean menuItemIsDisabled(String text)
    {
        if (findVisibleMenuItemOrNull(text) == null)
            throw new NoSuchElementException("Menu item not found: " + text);
        return findDisabledMenuItemOrNull(text) != null;
    }
```